app/models/rule.rb

```diff
@@ -11,7 +11,10 @@ class Rule < ApplicationRecord
   has_many :rule_results, dependent: :delete_all
   has_many :hosts, through: :rule_results, source: :host
 
+  validates :title, presence: true
   validates :ref_id, uniqueness: true, presence: true
+  validates :description, presence: true
+  validates :severity, presence: true
```

Comment:
Let's validate here if we're going to fail at the graphql layer with nil attributes.